README.md

```diff
@@ -38,54 +34,52 @@ gov.au UI-Kit
 - [tags](http://uikit.apps.staging.digital.gov.au/packages/tags/tests/site/)
 - [text-inputs](http://uikit.apps.staging.digital.gov.au/packages/text-inputs/tests/site/)
 
-
 **[⬆ back to top](#content)**
 
-
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-
 ## Build
 
-To build the UI-Kit you have to install [lerna](https://github.com/lerna/lerna) globally and run.
+Clone the UI-Kit project:
 
 ```shell
+git clone https://github.com/govau/uikit.git
+```
+
+If you haven't already,
+[install Yarn](https://yarnpkg.com/en/docs/instal).
+
+Install [lerna](https://github.com/lerna/lerna) globally.
+
+Run lerna from your UI-KIt clone:
+
+```shell
+cd uikit
 yarn
 lerna init
 lerna bootstrap
 ```
 
-To add a new module run the scaffolding helper:
+### Add a module
+
+Run the scaffolding helper:
 
 ```shell
 npm run scaffolding
 ```
 
-_After the you filled out all the blanks and added your dependencies into your package.json make sure you run `lerna bootstrap` again._
+Respond to the prompts to add dependencies to the package.json file.
 
+Run `lerna bootstrap` again.
 
 **[⬆ back to top](#content)**
 
-
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-
-## Tests
-
-Visual tests have been build into each module and can be seen in either the `README.md` files of each module or in the [listing above](#modules).
-
-
-**[⬆ back to top](#content)**
-
-
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-
 ## License
 
 Copyright (c) Commonwealth of Australia.
-Licensed under [MIT](https://raw.githubusercontent.com/govau/uikit/master/LICENSE).
-
+Licensed under [MIT license](https://raw.githubusercontent.com/govau/uikit/master/LICENSE).
```

Comment:
`MIT` is a license ID (https://spdx.org/licenses/) so maybe add the word license to the end?
eg:
Licensed under [MIT](https://raw.githubusercontent.com/govau/uikit/master/LICENSE) license.